cmd/medius/images/push.go

```diff
@@ -281,3 +354,9 @@ func prepareTags(timestamp time.Time, registry string, entry *common.Entry, arti
 
 	return names
 }
+
+func cleanupArtifacts(artifacts []string) {
+	for _, file := range artifacts {
+		os.Remove(file)
```

Comment:
check for errors?

Reply:
I don't think there is possibility that an error will be occurred. This function is only called when there are artifacts that created by Medius and should be removed by the end of `Do(...)` execution.